package.json

```diff
@@ -15,20 +15,23 @@
 	"bugs": {
 		"url": "https://github.com/faucet-pipeline/faucet-pipeline/issues"
 	},
-	"main": "index.js",
+	"main": "lib/index.js",
 	"bin": {
 		"faucet": "bin/faucet"
 	},
 	"scripts": {
-		"test": "eslint --cache *.js bin/* && echo ✓"
+		"test": "npm-run-all --parallel lint tests",
+		"tests": "mocha test/test_*.js",
```

Comment:
Would `test:all` be less confusing/awkward here? Any other ideas?

Reply:
I would suggest:

* `lint` is `eslint --cache *.js bin/* && echo ✓`
* `mocha` is `mocha test/test_*.js`
* `test` is `npm-run-all --parallel lint tests`

Reply:
As discussed elsewhere, I've changed "tests" to "test:unit" (even though it might not be perfectly accurate).

Let's squash that commit when merging though.